SCons/Environment.xml

```diff
@@ -374,6 +377,12 @@ foo = Program('foo.c')
 AddPostAction(foo, Chmod('$TARGET', "a-x"))
 </example_commands>
 
+<para>
+If a <parameter>target</parameter> is an Alias target,
+<parameter>action</parameter> is associated with the
+action of the alias, if specified.
```

Comment:
Should alias be the Alias entity here?

Reply:
I can't find a wording of this that feels comfortable. Riffing a bit, how about:
```
If <parameter>target</parameter> is an Alias node with an associated action, the post action will be executed after the alias' action.
```